packages/driver/cypress/integration/e2e/testConfigOverrides.spec.js

```diff
@@ -377,6 +377,41 @@ describe('testConfigOverrides baseUrl @slow', () => {
   })
 })
 
+describe('cannot set read-only properties', () => {
+  it('throws if mutating read-only config with test configuration', () => {
+    try {
+      it('fails to set chromeWebSecurity', { cypressEnv: 'test', chromeWebSecurity: false }, () => {
+        return
+      })
+    } catch (err) {
```

Comment:
If an error isn't thrown, this test will still pass. Using `expect(...).to.throw(...)` would protect against that.